tests/integration/two-active-chatters.spec.holochain.js

```diff
@@ -0,0 +1,337 @@
+/* global it, describe, expect, beforeAll, afterAll */
+import 'regenerator-runtime/runtime.js'
+import wait from 'waait'
+import { v4 as uuidv4 } from 'uuid'
+import { orchestrator } from './setup/tryorama'
+import { waitForState, awaitZomeResult, findElementByText, findElementByClassandText, getElementProperty, registerNickname, beforeAllSetup, afterAllSetup } from './setup/helpers'
+import { TIMEOUT, WAITTIME } from './setup/globals'
+import { INSTALLED_APP_ID } from '@/consts'
+
+// TEMPORARY: Remove and reset this extended waittime after hc header issue is resolved
+const EXTENDED_WAITTIME = WAITTIME + 3000
+
+orchestrator.registerScenario('Two Active Chatters', async scenario => {
+  const callRegistry = {}
+  const callStats = async () => await aliceChat.call('chat', 'stats', { category: 'General' })
+  const newChannel = {
+    name: 'First Room',
+    entry: { category: 'General', uuid: uuidv4() }
+  }
+  
+  let aliceChat, bobboChat, page, closeServer, conductor, newPage, stats, newStats, channelInFocus
+  beforeAll(async () => {
+    const createPage = async () => await global.__BROWSER__.newPage();
+    // Note: passing in Puppeteer page function to instantiate pupeeteer and mock Browser Agent Actions
+    ({ aliceChat, bobboChat, page, closeServer, conductor } = await beforeAllSetup(scenario, createPage, callRegistry))
+    await wait(EXTENDED_WAITTIME)
+    // scenario setup:
+    console.log('Setting up default channel...')
+    channelInFocus = await aliceChat.call('chat', 'create_channel', newChannel)
+    console.log('Verifying initial stats...', channelInFocus)
+    stats = await awaitZomeResult(callStats, 90000, 10000)
```

Comment:
It was used for list calls to make sure we actually get a value returned that we can verify.  There was a while where the original entry creations in the dht took a while, so even if the data was created correctly, sometimes the value wouldn't be returned on the first get list call. I did then use it a bit loosely in a few other places to as a more consistent way to await the value (since we couldn't use the `waitForState` fn for calls not trigged at the ui level).  Nonetheless, this issue is actually resolved in hc, so I'll remove this one.